lib/overcommit/hook/pre_push/git_lfs.rb

```diff
@@ -0,0 +1,43 @@
+module Overcommit::Hook::PrePush
+  # Replaces the pre-push hook installed by `git lfs install` that is then moved aside by
+  # `overcommit --install`
+  #
+  # Enable this hook in .overcommit.yml like so:
+  #
+  #  PrePush:
+  #    GitLfs:
+  #      enabled: true
+  #      command: ['git', 'lfs', 'pre-push']
+  #
+  # @see https://github.com/github/git-lfs
+  class GitLfs < Base
+    # Overcommit expects you to override this method which will be called
+    # everytime your pre-commit hook is run.
+    def run
+      # Create two arrays to hold our error and warning messages.
+      error_lines = []
+      warning_lines = []
+
+      # Check if git-lfs executable exists
+      path_to_git_lfs = `command -v git-lfs`.strip
```

Comment:
`git-lfs`. `git lfs` is just a shim for `git-lfs`, since `git` picks up any external commands available on your path.